### PR TITLE
Allow node-pool args to be empty

### PR DESCRIPTION
**Pull request by Retler**
This PR adds the capability of ignoring empty node-pool arguments. Some background to motivate this:

The documentation doesn't mention that the --num-nodes string should not terminate with a `;` character. Therefore, when trying to use an argument like `--num-nodes:"name=pool-name;count=3;"` the command fails with an error, because the trailing `;` character results in an empty arg.

### CODE REVIEW

File: commands/kubernetes.go [hunk 1]
```diff
@@ -1721,6 +1721,9 @@ func parseNodePoolString(nodePool, defaultName, defaultSize string, defaultCount
 		Taints: []godo.Taint{},
 	}
 	for _, arg := range strings.Split(nodePool, argSeparator) {
```
Comment (andrewsomething) [inline commands/kubernetes.go:1723]: How about trimming any trailing `;`?

```go
	trimmedPool := strings.TrimSuffix(nodePool, argSeparator)
	for _, arg := range strings.Split(trimmedPool, argSeparator) {
```

It's a bit of a corner case, but it provides a proper error if someone were to run:

    doctl kubernetes cluster create test --node-pool ";"

**Review by andrewsomething — COMMENTED** [review]
Thanks for this change! I left a comment in-line with a suggestion.

File: commands/kubernetes.go (hunk 1 above)
Comment (Retler) [reply commands/kubernetes.go]: That seems reasonable. Thanks for the review, I have now modified the code.

**Review by andrewsomething — APPROVED** [review]
:+1:  LGTM; thanks again!

**Comment by MattIPv4** [comment]
Hey, @Retler - Thanks a ton for this sweet PR! 😄

Would you please shoot me an email when you get a chance?
mcowley at digitalocean dot com 🎉
